src/test/java/org/apache/commons/io/IOUtilsTestCase.java

```diff
@@ -1186,6 +1187,89 @@ private void testToString_URL(final String encoding) throws Exception {
         }
     }
 
+    @Test public void testResourceToString_ExistingResourceAtRootPackage() throws Exception {
+        final long fileSize = new File(getClass().getResource("/test-file-utf8.bin").getFile()).length();
+        final String content = IOUtils.resourceToString("/test-file-utf8.bin", Charset.defaultCharset());
```

Comment:
Charset.defaultCharset here and in other tests seems wrong as it is locale/environment dependent. StandardCharsets.UTF_8 may be more appropriate.